artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -429,8 +444,28 @@ public String getSystemPropertyPrefix() {
       return systemPropertyPrefix;
    }
 
+   public String getBrokerPropertiesSystemPropertyName() {
+      return brokerPropertiesSystemPropertyName;
+   }
+
+   public void setBrokerPropertiesSystemPropertyName(String brokerPropertiesSystemPropertyName) {
+      this.brokerPropertiesSystemPropertyName = brokerPropertiesSystemPropertyName;
+   }
+
    @Override
-   public Configuration parseSystemProperties() throws Exception {
+   public Configuration parseProperties() throws Exception {
+      final String propertiesConfigFileName = System.getProperty(getBrokerPropertiesSystemPropertyName());
```

Comment:
I think we should pass this through the CLI instead.

We could parse a default broker.properties on the instance/etc folder, and the users would be able to pass another file with a ./artemis run --properties <file>

Reply:
I am not sure of the benefit of multiple files, but the cli could arrange for the system property to be initialised with a value from a --properties argument for sure.

Reply:
@gtully I didn't mean to ask for multiple properties file.. just the cli argument